canonical/ubuntu.com
# WD-12357 Created new form for /containers

## Done

- New form for /containers webpage for CTA at the top ‘Get a secure, custom-built container’
## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Please see if all the columns are as per [copydoc](https://docs.google.com/document/d/14DLnRamcE2su06Zh4zi9sEwJ_dpQFljH-iOVdKxjc7U/edit)
- Here is the [demo](https://ubuntu-com-14034.demos.haus/containers#get-in-touch) for the form.

## Issue / Card

Fixes # [WD-12357](https://warthogs.atlassian.net/browse/WD-12357)

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)


[WD-12357]: https://warthogs.atlassian.net/browse/WD-12357?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
